langchain-core/src/messages/base.ts

```diff
@@ -195,7 +208,12 @@ export abstract class BaseMessage
       // eslint-disable-next-line no-param-reassign
       fields.response_metadata = {};
     }
+    if (!fields.id) {
+      // eslint-disable-next-line no-param-reassign
+      fields.id = undefined;
```

Comment:
This seems unnecessary?

Reply:
This is necessary.  Otherwise ids added  via mutation won' tbe serialized